server.R

```diff
@@ -74,7 +74,13 @@ shinyServer(function(input, output, session) {
     syn_login(authToken = access_token, rememberMe = FALSE)
 
     # updating syn storage
-    tryCatch(syn_store <<- synapse_driver(access_token = access_token), error = function(e) NULL)
+    syn_store <<- tryCatch(
+      synapse_driver(access_token = access_token),
+      error = function(e) {
+        message(e$message)
```

Comment:
@rchai Just confirming, these error messages are only printed to the logs and not shown to the end user, right? In general, error messages sometimes expose information that hackers may use; or at least would be confusing to users , so we want to avoid surfacing that :)